### ISSUE TITLE
Unable to manually add covers to USB or SD

### ISSUE THREAD

**Issue by alvv-z** [Fledge68/WiiFlow_Lite#251]
Hi all, my Wii's WIFI doesnt work with my router and I have been trying to get manual covers working by placing the pngs in `sd:/wiiflow/boxcovers` or `usb:/wiiflow/boxcovers` with their titleID from gametdb as the name + `.png`. I then changed the names of my games to their IDs, and it still wont work. I'm not sure if this is a bug or an error on my part. Thank you for the help.

**Comment by gingerbeardman** [comment]
The covers need to be in a special format for WFL, normal operation it will download them and convert them before saving.

Can you connect to wifi elsewhere (phone hotspot?) just to grab the covers?

**Comment by alvv-z** [comment]
No, for some reason I can't connect to any of my access points (search won't show my home WiFi or my phone, and manual connection won't work either) even though other access points are being shown, so I have to transfer them manually.

**Comment by Fledge68** [comment]
after you add new covers manually you need to view your current covers then go to HOME MENU to select RELOAD CACHE. if it doesn't work you have them in the wrong folder or named wrong.

**Comment by alvv-z** [comment]
How would I have to name them? I have tried titleID, wbfs name, and wbfs name with extension and none seem to work.
